lib/puppet.rb

```diff
@@ -74,7 +74,7 @@ class << self
       end
 
       if locale_path
-        if Gem.loaded_specs['gettext-setup'].version < Gem::Version.new('0.8')
+        if Gem::Version.new(GettextSetup::VERSION) < Gem::Version.new('0.8')
```

Comment:
Perhaps we can check if `GettextSetup` has the _functionality_ that is required, rather than checking for the version number.